From 86ed671c8efbe4b5c5e51a23cdc207901e7bd014 Mon Sep 17 00:00:00 2001
From: Emmanuel Gil Peyrot <linkmauve@linkmauve.fr>
Date: Thu, 26 Nov 2020 23:57:01 +0100
Subject: [PATCH 3/9] =?UTF-8?q?Give=20str=20its=20Python=C2=A03=20meaning?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Cython still defaults to aliasing it to bytes for compatibility with
Python 2, but we don’t need that anymore.
---
 mpv.pyx | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mpv.pyx b/mpv.pyx
index 8cef63d..c353493 100644
--- a/mpv.pyx
+++ b/mpv.pyx
@@ -1,3 +1,5 @@
+# cython: language_level=3
+
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation, either version 3 of the License, or
-- 
2.40.1

